### PR TITLE
fix: adaptive-expressions-ie11 webpack config

### PR DESCRIPTION
**Pull request by joshgummersall**
- Use `require.resolve` for proper package resolution algorithm, do not
hardcode paths to packages
- Use `CleanWebpackPlugin`
- Use `CopyWebpackPlugin` instead of `copyfiles` for proper package
resolution algorithm

Replaces #3087

### CODE REVIEW

**Comment by joshgummersall** [comment]
@atishayv please review this when you get a chance.

File: libraries/adaptive-expressions-ie11/package.json [hunk 1]
```diff
@@ -31,17 +31,15 @@
...
+    "clean-webpack-plugin": "^3.0.0",
     "terser-webpack-plugin": "^4.2.3",
     "ts-loader": "^7.0.5",
     "typescript": "3.5.3",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.12"
   },
   "scripts": {
-    "build:webpack": "webpack",
-    "build:copyfiles": "copyfiles -u 3 node_modules/adaptive-expressions/lib/**/*.d.ts dist",
```
Comment (joshgummersall) [inline libraries/adaptive-expressions-ie11/package.json:43]: Note: the generated declaration file is:

```typescript
export * from 'adaptive-expressions';
```

Which the Typescript compiler will properly resolve. No need to copy over all the generated declaration files.

**Comment by atishayv** [comment]
> @atishayv please review this when you get a chance.

Thanks for fixing my PR. The changes looks good to me

**Review by stevengum — APPROVED** [review]
LGTM, since @atishayv signed off... :shipit:

**Pull request merged**